apps/admin/sbox/00/aad-pod-id-patch.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nmi
+spec:
+  tolerations:
+  - effect: NoSchedule
+    key: CriticalAddonsOnly
+    operator: Equal
+    value: "true"
+  affinity:
+  nodeAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - weight: 1
```

Comment:
Lets see if this works, can we discuss this? (I did ask for this nmi to be added :D )

I am unsure if we want DaemonSet included - @hannah38 lets all catchup tomorrow.